src/sfizz/Region.cpp

```diff
@@ -92,7 +92,11 @@ bool sfz::Region::parseOpcode(const Opcode& rawOpcode)
             offsetCC[opcode.parameters.back()] = *value;
         break;
     case hash("end"):
-        setValueFromOpcode(opcode, sampleEnd, Default::sampleEndRange);
+        if (opcode.value == "-1") {
+            disabled = true;
+        } else {
+            setValueFromOpcode(opcode, sampleEnd, Default::sampleEndRange);
+        }
```

Comment:
We probably want to re-enable the region if sampleEnd is reset to a positive value after -1.
`<region>end=-1 end=10000`

Here I don't think the `disabled` boolean to be a great idea, I'd rather have `bool disabled() const` to check for `sampleEnd` where the member `sampleEnd` is changed to an optional.

Btw does `end` not also disable the voice if 0 ? I recall it to be the case.
Hence we will want to check `disabled` = `end` is set, and is <= 0.

Reply:
Ok then I guess the best is to make it an int64, and create this `disabled()` function.

Reply:
In the end I chose to keep the type. ARIA complains when sampleEnd==0 but still disables the region, so we can just do the same.

Otherwise I guess it'd be better to audit the uses of int64/uint32 in file and loop lengths. libsndfile uses int64 for the file length but the wav chunks are uint32.